app/helpers/reactjs_helper.rb

```diff
@@ -1,8 +1,25 @@
 module ReactjsHelper
+  def react_component(name, opts = { data: [], flatten_data: false })
+    data = opts[:data]
+    data = data.to_json if data.is_a?(Hash)
+
+    attributes = {
+      name: name,
+      'data-props': data,
+    }
+
+    attributes[:flatten] = true if opts[:flatten_data]
```

Comment:
Do you think it makes the attribute descriptive enough?
I chose to stick with `flatten-data` in my version, but I would be OK to use `flatten` if you think. It is easier to use :) :man_shrugging:

Reply:
Agree `flatten-data` would be a better naming, will change it.